linter.py

```diff
@@ -35,6 +34,20 @@ class Flow(Linter):
 
     __flow_near_re = '`(?P<near>[^`]+)`'
 
+    def run(self, cmd, code):
+        """
+        flow lint code if `@flow` pragma is present
```

Comment:
It silly but would you mind adding a period after `present` so that the liter passes?